### DISCUSSION TITLE
The Sneaky Triangle

### DISCUSSION THREAD

**Discussion by meviru** [meviru/badge-hunting#15] [Galaxy Trivia] [answered]
Your element wants to cosplay as a triangle for Halloween. How can you help it transform using only CSS?

**Comment by mitchiemt11** [comment, marked as answer]
@meviru Assuming I understand the task correctly, we want to transform an HTML element into a triangle shape. To achieve this, we can use CSS borders with the following approach:

- Set the `width` and `height` of the element to 0 to remove any default sizing.
- Set the `border-left` and `border-righ`t properties to equal values and make them transparent to form the sides of the triangle.
- Use the `border-bottom `property to define the visible part of the triangle, including its height and color.

**Reply by meviru** [reply]
Correct!! It would be great if you could also provide an example for the community.

**Reply by meviru** [reply]
Congratulations!! 👏👏 You've just earned the [Galaxy Brain](https://github.com/mitchiemt11?tab=achievements&achievement=galaxy-brain) badge. Keep it up, and remember to give the repository a star to show your support! 🎉

**Comment by PranjalTrivedi** [comment]
Mark as Answer
